docs/bigquery/snippets.py

```diff
@@ -2297,11 +2297,31 @@ def test_client_list_jobs(client):
     # [START bigquery_list_jobs]
     # from google.cloud import bigquery
     # client = bigquery.Client(project='my_project')
+    import datetime
 
     # List the 10 most recent jobs in reverse chronological order.
     # Omit the max_results parameter to list jobs from the past 6 months.
+    print("Last 10 jobs:")
     for job in client.list_jobs(max_results=10):  # API request(s)
         print(job.job_id)
+
+    # The following are examples of additional optional parameters:
+
+    # Use min_creation_time and/or max_creation_time to specify a time window.
+    two_days_ago = datetime.datetime.now() - datetime.timedelta(days=2)
```

Comment:
I think we may want to use the `utcnow()` function instead of `now()`. I believe `now()` can return the local time but doesn't always populate the local time zone correctly.

Reply:
From https://julien.danjou.info/python-and-timezones/

```
>>> import datetime
>>> datetime.datetime.utcnow()
datetime.datetime(2015, 6, 15, 13, 24, 48, 27631)
>>> datetime.datetime.now()
datetime.datetime(2015, 6, 15, 15, 24, 52, 276161)
```

> As you can notice, none of these results contains timezone information. Indeed, Python datetime API always returns unaware datetime objects,